javatests/arcs/android/storage/database/DatabaseImplTest.kt

```diff
@@ -3015,6 +3015,43 @@ class DatabaseImplTest {
         assertThat(exception).hasMessageThat().isEqualTo("Key number is missing in the map.")
     }
 
+    @Test
+    fun fails_SecondWrite_WhenSchemas_Overlap() = runBlockingTest {
```

Comment:
instead of Overlap, you could be more precise eg fails_SecondWrite_WhenDifferentSchemasHaveSameHash